### DISCUSSION TITLE
One of debbugs info files missing

### DISCUSSION THREAD

**Discussion by spepo** [emacscollective/borg#122] [General]
Hi Jonas, 
After assimilating debbugs package I discovered that one of the info files was missing (debbugs-ug.info) in my Info manual. Looking at borg's build output everything looked fine -- no errors reported. However the info file is missing. Re-running the makeinfo command showed the following error:
```
/Users/spepo42/emacs.d/lib/debbugs//debbugs-ug.texi:823: `Customizing debbugs-gnu-pick-commits' has no Up field (perhaps incorrect sectioning?).
/Users/spepo42/emacs.d/lib/debbugs//debbugs-ug.texi:823: warning: unreferenced node `Customizing debbugs-gnu-pick-commits'.
makeinfo: Removing output file `debbugs-ug.inf' due to errors; use --force to preserve.
```
With the --force option it went through (although the mentioned section is missing in the manual).

Do you see the same problem with debbugs-ug.texi?  Obviously, I should report the issue to debbugs maintainers.

But could borg try to recover automagically and attempt to redo the build in such situations with --force? Or just report the error in `Borg Build` buffer?

Thanks,
Peter

**Comment by tarsius** [comment]
> Or just report the error in `Borg Build` buffer?

I'm probably gonna do that.
